plugin/src/main/java/io/jenkins/plugins/remotingkafka/kafkaclient/KafkaConsumerClient.java

```diff
@@ -0,0 +1,57 @@
+package io.jenkins.plugins.remotingkafka.kafkaclient;
+
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+
+import java.util.List;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class KafkaConsumerClient {
+    private static final Logger LOGGER = Logger.getLogger(KafkaConsumerClient.class.getName());
+    private static KafkaConsumerClient instance;
```

Comment:
If the code is used in a plugin, it is better to declare a singleton via `@Extension` annotation. Otherwise it may cause issues, e.g. for JUnit tests. Generally we try to avoid static fields in plugins where possible